### DISCUSSION TITLE
Is this the correct schema?

### DISCUSSION THREAD

**Discussion by nogeenkopjethee** [wp-graphql/wp-graphql#1628] [General]
I've been trying to export the schema and read it out with Apollo Client, but I get a schema like this:

https://gist.github.com/nogeenkopjethee/ca708125c5a0fe2ace816e7c1a0c52b6

where everything is deprecated, instead of only the deprecated items.

**Comment by jasonbahl** [comment]
@nogeenkopjethee this was a bug and it was resolved recently. 

See: https://github.com/wp-graphql/wp-graphql/pull/1633
